src/analyticsdf/analyticsdataframe.py

```diff
@@ -335,3 +335,22 @@ def update_response_poly_categorical(self, predictor_name: str = None, betas: di
                 self.response_vector = numeric_vals[predictor_name]
             else:
                 self.response_vector += numeric_vals[predictor_name]
+
+    def update_predictor_bernoulli(self, predictor_name_list,
+                               prob):
+        """Update the predictors of the instance to bernoulli distributed.
+
+        Args:
+            predictor_name_list:
+                A list of predictor names in the initial AnalyticsDataframe.
+            prob:
+                float or array_like of floats, probability of success
+           
+        Raises:
+            KeyError: If the column does not exists.
+        """
+        with set_random_state(validate_random_state(self.seed)):
+            num_row = len(self.predictor_matrix)
+            for predictor_name in predictor_name_list:
```

Comment:
Please check the user input predictor is correct using `_check_columns_exist()` function